### ISSUE TITLE
Next.js on windows crashes on uncaught syntax error

### ISSUE THREAD

**Issue by littledivy** [denoland/deno#21645]
```
$  ..\deno\target\debug\deno.exe run -A --unstable-byonm .\node_modules\.bin\next

error: Uncaught SyntaxError: missing ) after argument list
    at Object.evalContext (ext:core/01_core.js:826:14)
    at wrapSafe (node:module:704:25)
    at Module._compile (node:module:718:27)
    at Object.Module._extensions..js (node:module:745:10)
    at Module.load (node:module:656:32)
    at Function.Module._load (node:module:537:12)
    at Object.loadCjsModule (ext:deno_node/02_init.js:49:22)
    at loadCjsModule ([ext:ext\node\lib.rs:631:29]:2:32)
    at [ext:ext\node\lib.rs:631:29]:3:7
```

**Comment by littledivy** [comment]
Fixed itself as of v1.40.3

**Comment by vcha0018** [comment]
Hi @littledivy, I know it's too late, but can you mention how you fix this?
